subsys/net/ip/l2/ieee802154/ieee802154_utils.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2017 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief IEEE 802.15.4 Management
+ */
+
+#ifndef __IEEE802154_UTILS_H__
+#define __IEEE802154_UTILS_H__
+
+#include <net/ieee802154_radio.h>
+
+static inline void ieee802154_filter_ieee_addr(struct net_if *iface,
+					       u8_t *ieee_addr)
```

Comment:
Why are these functions inlined?

Reply:
Most of them are used in one unique place (only ieee_addr is used twice actually). For the other radio function (rx/tx/etc...), they are going to be one liners. I don't think it's worth having them exposed another way.